src/main/java/core/basesyntax/FigureDraw.java

```diff
@@ -0,0 +1,5 @@
+package core.basesyntax;
+
+public interface FigureDraw {
```

Comment:
The interface name 'FigureDraw' is a verb phrase, which is not recommended for class or interface names. Consider renaming it to something that represents the capability or role, like 'Drawable' or 'FigureRenderer'.